### PR TITLE
Add information about JSoC

### PR DESCRIPTION
**Pull request by shashi**
Edit as you see fit!

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ layout: default
 title:  The Julia Language
 ---
 
+**News:** 8 projects accepted in [Julia Summer of Code](http://julialang.org/jsoc.html)
```
Comment (johnmyleswhite) [inline index.md]: Do we usually announce news like this?

**Comment by johnmyleswhite** [comment]
LGTM. Not sure about changing the front page.

**Comment by ViralBShah** [comment]
The News could be a blog post, right?

**Comment by jiahao** [comment]
Closes #246

File: jsoc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title:  Julia Summer of Code
+---
+
+## Julia Summer of Code 2015
+
+This year Julia is organizing its own Summer of Code there are 8 different projects under the program.
```
Comment (mlubin) [inline jsoc.md]: Run-on sentence here

**Pull request merged**